src/domain/Contact.ts

```diff
@@ -445,7 +445,7 @@ export default class Contact {
     }
 
     return new Contact({
-      name: `${plain.givenName || ''} ${plain.familyName || ''}`,
+      name: `${plain.givenName || plain.company || ''} ${plain.familyName || ''}`,
```

Comment:
Can you please use `company` only if `givenName` and `familyName` are empty ?

Reply:
Added